programming/python/index.md

```diff
@@ -15,11 +15,11 @@ There are a number of tutorials out there which might help you to learn to progr
     This was written especially for Student Robotics competitors,
     and explains the basics while trying not to overwhelm you.
 
-* [The Official Tutorial](https://docs.python.org/3.9/tutorial/) -- the good stuff starts at chapter 3,
+* [The Official Tutorial](https://docs.python.org/3.10/tutorial/) -- the good stuff starts at chapter 3,
     but you should at least skim the stuff before it;
     there is a lot there and it may be a little overwhelming.
 
 * A number of tutorials for beginners are linked to from [here](https://wiki.python.org/moin/BeginnersGuide/NonProgrammers).
     [Wikibooks Tutorial](https://en.wikibooks.org/wiki/Non-Programmer%27s_Tutorial_for_Python_3) seems like a good one.
 
-* [The Official Docs](https://docs.python.org/3.9/), for the version of python on the [Odroid Brain Boards](/docs/kit/brain_board).
+* [The Official Docs](https://docs.python.org/3.10/), for the version of python on the [Odroid Brain Boards](/docs/kit/brain_board).
```

Comment:
Should we be talking about Pis here now?

Reply:
Yeah, I just noticed that too. We should, though I think there's a separate task for that (it involves a deeper change to the brain board docs). Happy to hold this change and let it be part of that PR though.

Reply:
Let's hold this change until later. 👍

Reply:
rebased on #355.